internal/wrappers/feature-flags.go

```diff
@@ -98,12 +145,17 @@ func loadFeatureFlagsDefaultValues() {
 
 type FeatureFlagsWrapper interface {
 	GetAll() (*FeatureFlagsResponseModel, error)
+	GetSpecificFlag(specificFlag string) (*FeatureFlagResponseModel, error)
 }
 
 type FeatureFlagsResponseModel []struct {
```

Comment:
you can use the new single flag struct as array - you don't need the 2 structs

Reply:
suggestion, cause it will need tests

Reply:
Makes sense